src/AddTodoForm.jsx

```diff
@@ -1,22 +1,30 @@
-const AddTodoForm = (props) => {
+import { useState } from 'react';
+
+const AddTodoForm = ({onAddTodo}) => {
+    let [todoTitle, setTodoTitle] = useState("");
+
+    const handleTitleChange = (event) => {
+        let todoInput = document.querySelector("input");
+        let newTodoTitle = todoInput.value;
+        //let newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+    };
+
     const handleAddTodo = (event) => {
+        // let form = event.target;
         event.preventDefault();
-        console.log("Form Submitted!");
-        let form = event.target;
+        onAddTodo({id: Date.now(), title: todoTitle});
+        setTodoTitle("");
         let todoInput = document.querySelector("input");
-        let todoTitle = todoInput.value;
-        //let todoTitle = event.target.value;
-        //console.log(form);
-        //console.log(todoInput);
-        console.log(todoTitle);
-        form.reset();
-        props.onAddTodo(todoTitle);
-    }
+        todoInput.value = "";
+        // console.log("Todo input field's value:", todoInput.value);
+        // console.log("todoTitle state;", todoTitle);
+    };
 
     return (
         <form onSubmit={handleAddTodo}>
             <label htmlFor="todoTitle">Title</label>&nbsp;&nbsp;
-            <input type="text" name="title" id='todoTitle' />&nbsp;&nbsp;
+            <input type="text" name="title" id='todoTitle' value={todoTitle} onChange={handleTitleChange} />&nbsp;&nbsp;
```

Comment:
Observation- As we get on into later weeks, we will be working with CSS which will help out with the spacing issue that you are addressing right here with `&nbsp;&nbsp;`.

Reply:
Meanwhile, is it okay if I continue to use &nbsp;&nbsp; for aesthetic reasons? I mean, just until I learn how to use CSS with React for styling purposes!